client/src/main/scala/drt/client/components/ToolTips.scala

```diff
@@ -55,8 +55,8 @@ object ToolTips {
 
   val totalPaxTooltip = Tippy.info(<.div(<.p("RAG colours are used to indicate DRT's confidence levels in the accuracy of its passenger number forecasts. Confidence levels depend on the source of data being displayed."),
     <.p("green -> live API or Port data is available for the flight and we have high confidence levels"),
-    <.p("amber-> We derive from historic API data"),
-    <.p("red -> We rely on ACL Forecast data")))
+    <.p("amber-> either derived from historic API data or port forecast pax is available"),
```

Comment:
`from port forecast when available or else derived from historic API data'

Reply:
actually to keep in line with existing descriptions, `Port forecast is available or historic API data is used as a fallback`
I think we might want a new ticket to change these descriptions, and I'm not sure about using amber for both a port forecast and our own estimates, but fine for now